hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -49,120 +49,128 @@
+        0x20L, // 1 - max(0,x)     OK for final combiner
+    PS_INPUTMAPPING_EXPAND_NORMAL =
+        0x40L, // 2*max(0,x) - 1   invalid for final combiner
+    PS_INPUTMAPPING_EXPAND_NEGATE =
+        0x60L, // 1 - 2*max(0,x)   invalid for final combiner
+    PS_INPUTMAPPING_HALFBIAS_NORMAL =
+        0x80L, // max(0,x) - 1/2   invalid for final combiner
+    PS_INPUTMAPPING_HALFBIAS_NEGATE =
+        0xa0L, // 1/2 - max(0,x)   invalid for final combiner
+    PS_INPUTMAPPING_SIGNED_IDENTITY =
+        0xc0L, // x                invalid for final combiner
+    PS_INPUTMAPPING_SIGNED_NEGATE =
+        0xe0L, // -x               invalid for final combiner
 };
 
-enum PS_REGISTER
-{
-    PS_REGISTER_ZERO=              0x00L, // r
-    PS_REGISTER_DISCARD=           0x00L, // w
-    PS_REGISTER_C0=                0x01L, // r
-    PS_REGISTER_C1=                0x02L, // r
-    PS_REGISTER_FOG=               0x03L, // r
-    PS_REGISTER_V0=                0x04L, // r/w
-    PS_REGISTER_V1=                0x05L, // r/w
-    PS_REGISTER_T0=                0x08L, // r/w
-    PS_REGISTER_T1=                0x09L, // r/w
-    PS_REGISTER_T2=                0x0aL, // r/w
-    PS_REGISTER_T3=                0x0bL, // r/w
-    PS_REGISTER_R0=                0x0cL, // r/w
-    PS_REGISTER_R1=                0x0dL, // r/w
-    PS_REGISTER_V1R0_SUM=          0x0eL, // r
-    PS_REGISTER_EF_PROD=           0x0fL, // r
-
-    PS_REGISTER_ONE=               PS_REGISTER_ZERO | PS_INPUTMAPPING_UNSIGNED_INVERT, // OK for final combiner
-    PS_REGISTER_NEGATIVE_ONE=      PS_REGISTER_ZERO | PS_INPUTMAPPING_EXPAND_NORMAL,   // invalid for final combiner
-    PS_REGISTER_ONE_HALF=          PS_REGISTER_ZERO | PS_INPUTMAPPING_HALFBIAS_NEGATE, // invalid for final combiner
-    PS_REGISTER_NEGATIVE_ONE_HALF= PS_REGISTER_ZERO | PS_INPUTMAPPING_HALFBIAS_NORMAL, // invalid for final combiner
+enum PS_REGISTER {
+    PS_REGISTER_ZERO = 0x00L, // r
+    PS_REGISTER_DISCARD = 0x00L, // w
+    PS_REGISTER_C0 = 0x01L, // r
+    PS_REGISTER_C1 = 0x02L, // r
+    PS_REGISTER_FOG = 0x03L, // r
+    PS_REGISTER_V0 = 0x04L, // r/w
+    PS_REGISTER_V1 = 0x05L, // r/w
+    PS_REGISTER_T0 = 0x08L, // r/w
+    PS_REGISTER_T1 = 0x09L, // r/w
+    PS_REGISTER_T2 = 0x0aL, // r/w
+    PS_REGISTER_T3 = 0x0bL, // r/w
+    PS_REGISTER_R0 = 0x0cL, // r/w
+    PS_REGISTER_R1 = 0x0dL, // r/w
+    PS_REGISTER_V1R0_SUM = 0x0eL, // r
+    PS_REGISTER_EF_PROD = 0x0fL, // r
+
+    PS_REGISTER_ONE = PS_REGISTER_ZERO |
+                      PS_INPUTMAPPING_UNSIGNED_INVERT, // OK for final combiner
+    PS_REGISTER_NEGATIVE_ONE =
+        PS_REGISTER_ZERO |
+        PS_INPUTMAPPING_EXPAND_NORMAL, // invalid for final combiner
+    PS_REGISTER_ONE_HALF =
+        PS_REGISTER_ZERO |
+        PS_INPUTMAPPING_HALFBIAS_NEGATE, // invalid for final combiner
+    PS_REGISTER_NEGATIVE_ONE_HALF =
+        PS_REGISTER_ZERO |
+        PS_INPUTMAPPING_HALFBIAS_NORMAL, // invalid for final combiner
```

Comment:
Liked the old formatting better on this block. At least we should attempt to *always* have the `_ZERO` in either the first or second line, but not mixing it.